### ISSUE TITLE
Python 3.6 Requirement Not Mentioned

### ISSUE THREAD

**Issue by jobe1986** [GregoryAM-SP/The-Minecraft-Overviewer#6]
This may sound like a strange thing to point out but since you've used one single f'' string for formatting in your updates you have increased the minimum requirement to Python 3.6, Therefore I feel like this minimum requirement should be listed somewhere at least.

Whilst I understand Python 3..5 and earlier are no longer officially supported, they are still installed in some cases, as is the case for me.

Alternatively, since the f'' syntax has only been used in one single place so far, it wouldn't hurt to use a syntax that's compatible with old versions too. Also it has been noted that concatenation (use of +) is slightly faster (not by much though) performance wise then f-strings, though not enough to notice unless you're formatting a large enough quantity of data.

Anyway in summary, seeing as it's not currently mentioned, and overviewer.org says only that Python 3 is required, and not any specific minimum version, it would be appropriate if this limitation was mentioned at least in the readme.

Also, thank you for your efforts.

**Comment by Gregory-AM** [comment]
That's strange.
Can you point out the code that contains the f" string format?
Maybe I can rework it and change it to support older versions of Python.

**Comment by Gregory-AM** [comment]
@jobe1986, I understand this was a month ago.
But I've recently released Overviewer using Python 3.8.

**Comment by jobe1986** [comment]
> @jobe1986, I understand this was a month ago. But I've recently released Overviewer using Python 3.8.

Thanks for the update, and just as an FYI, because I like to torture myself, the code as of commit 718fe57 (Latest release) still runs on Python 3.5 with only this single modification:
```
diff --git a/overviewer_core/world.py b/overviewer_core/world.py
index 64304f9..387af55 100644
--- a/overviewer_core/world.py
+++ b/overviewer_core/world.py
@@ -1344,7 +1344,7 @@ class RegionSet(object):
             data = {'south': 0, 'west': 1, 'north': 2, 'east': 3}[facing]
             p = palette_entry['Properties']
             for i in range(6):
-                if p[f'slot_{i}_occupied'] == 'true':
+                if p['slot_'+str(i)+'_occupied'] == 'true':
                     data |= 4
         elif (key in 'minecraft:calibrated_sculk_sensor'):
             facing = palette_entry['Properties']['facing']

```
